Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -667,17 +669,81 @@ class BrowserViewController: UIViewController {
     }
     
     func presentOnboardingIntro() {
+        let isRewardsEnabled = rewards?.ledger.isEnabled == true
+        
+        // 1. Existing user.
+        // 2. The user skipped onboarding before.
+        // 3. 60 days have passed since they last saw onboarding.
+        if Preferences.General.basicOnboardingCompleted.value == OnboardingState.skipped.rawValue {
+
+            guard let daysUntilNextPrompt = Preferences.General.basicOnboardingNextOnboardingPrompt.value else {
+                return
+            }
+            
+            // 60 days has passed since the user last saw the onboarding.. it's time to show the onboarding again..
```

Comment:
Is the 60 days for basic onboarding as well?

Reply:
It's actually not clarified if it's for basic or not but it does say:

"If the user SKIPS" the on-boarding, we show them again in 60 days. I'm not sure if this applies to search engine but I can't see why it won't since it's also onboarding. I'll ask.

Reply:
This matches Android, and we will be sticking with that for now